### DISCUSSION TITLE
Cargo: command not found

### DISCUSSION THREAD

**Discussion by IndecisivePotato** [bevyengine/bevy#3354] [Q&A] [answered]
Hello, just started with Bevy - Getting Started, downloaded Git Bash, made first few step and on "_Try the examples in the examples folder
       cargo run --example breakout_"
i see an error "_bash: cargo: command not found_"
Can't seem to find what the issue is, pls help

**Comment by DJMcNab** [comment, marked as answer]
Hmm that's strange. As you followed the setup guide, you should have Rust installed.

When you installed rust from https://rustup.rs/ as instructed in the [setup guide](https://bevyengine.org/learn/book/getting-started/setup/), did you select the option to the cargo home to `PATH`?

**Reply by IndecisivePotato** [reply]
It worked! I didn't have Rust installed... And it was right on the next page. Thank you!
